tracpro/settings/base.py

```diff
@@ -333,7 +333,7 @@ def _org_scheduler_task(task_name):
         'delete',  # can delete an object
         'list',  # can view a list of the objects
     ),
-    'orgs.org': ('create', 'update', 'list', 'edit', 'home'),
+    'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'fetch_runs'),
```

Comment:
Ahh I remember running into this previously - what you've got here creates a permission called `orgs.org_fetch_runs`. Smartmin insists on checking that there's an app called `orgs` and a model called `org`. 

So I think you have to use `orgs.org_fetch_runs` as your permission on the view